plugins/katello/nightly/user_guide/content_hosts/index.md

```diff
@@ -166,10 +172,20 @@ Next select the 'Bulk Actions' button in the top right.
 
 From here you can select the tab corresponding to any action you wish to perform.
 
-## Set System Purpose attributes
+### Setting System Purpose attributes via Bulk Actions
 
-To set System Purpose attributes for a host, Navigate to the Content Host Details page,  Host > Content Hosts > Click the name of the desired Content Host. You can edit Service Level, Role, Usage Type, and Add-ons from the System Purpose section.
+Select the Content Hosts that you wish to change the System Purpose attributes on.
 
-![System Purpose](/plugins/katello/{{ page.version }}/user_guide/content_hosts/system_purpose.png)
+Next select the 'Bulk Actions' button in the top right, and choose 'Manage System Purpose'
+
+In the following screen, a banner at the top displays how many Content Hosts will be affected by the change. Select the dropdown next to the attributes you want to change, select the desired value and when satisfied with the options, click the 'Assign' button.
+
+![Bulk System Purpose](/plugins/katello/{{ page.version }}/user_guide/content_hosts/bulk_system_purpose.png)
+
+**Some following things to note about how the behaivor works with the dropdowns:**
```

Comment:
```suggestion
**Note**
```

Reply:
(I was going to call out the "behavior" typo but I think the single word is better)

Reply:
Updated.